### PR TITLE
remove de_json()

### LINKED ISSUE #778: Remove de_json from object that never use them
We have a lot of objects in the library that will only ever be sent, never received. But they still do have a de_json method. I propose to remove all de_json methods from objects that are never sent by tg. Below you'll find a list of the objects involved

- [x] Bot
- [x] ForceReply
- [x] InlineKeyboardButton
- [x] InlineKeyboardMarkup
- [x] InlineQueryResult*
- [x] InlineQueryResultCached*
- [x] InputMessageContent
- [x] Input*MessageContent
- [x] KeyboardButton
- [x] LabeledPrice
- [x] ReplyKeyboardMarkup
- [x] ReplyKeyboardRemove
- [x] ShippingOption

### PR DESCRIPTION
**Pull request by Eldinnie**
Remove unneeded and unused de_json and de_list methods from our library. The only usages for these methods in our lib were within each other's de_json methods and our tests.
Expect Travis to fail, let it run again when #788 is merged.

Fixes #778

### CODE REVIEW

**Comment by Eldinnie** [comment]
tests passed. review wanted

File: tests/test_inputtextmessagecontent.py [hunk 1]
```diff
@@ -43,25 +34,10 @@ class TestInputTextMessageContent(object):
...
-
-        assert isinstance(input_text_message_content_json, InputTextMessageContent)
-
-    def test_de_json_factory_without_required_args(self, json_dict, bot):
-        del (json_dict['message_text'])
-
-        input_text_message_content_json = InputMessageContent.de_json(json_dict, bot)
-
-        assert input_text_message_content_json is None
+    def test_exspected_values(self, input_text_message_content):
```
Comment (bomjacob) [inline tests/test_inputtextmessagecontent.py]: Spelling error

File: tests/test_inputlocationmessagecontent.py [hunk 2]
```diff
@@ -19,46 +19,22 @@
...
-        'latitude': TestInputLocationMessageContent.latitude,
-    }
+from telegram import InputLocationMessageContent
 
 
 @pytest.fixture(scope='class')
 def input_location_message_content():
-    return InputLocationMessageContent(TestInputLocationMessageContent.longitude,
-                                       TestInputLocationMessageContent.latitude)
+    return InputLocationMessageContent(longitude=TestInputLocationMessageContent.longitude,
```
Comment (bomjacob) [inline tests/test_inputlocationmessagecontent.py]: Why did you add these as kwargs?
Throughout the rest of the test we pass as an arg if it's a arg and kwarg if kwarg.

Comment (Eldinnie) [inline tests/test_inputlocationmessagecontent.py]: because they were in the wrong order and adding kwargs was lazier then reversing the order. Will fix though

**Comment by Eldinnie** [comment]
@bomjacob I addressed your review changes.
